cdap-support-bundle/bin/collect_support_bundle.py

```diff
@@ -0,0 +1,660 @@
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+# collect_support_bundle.py to get logs from CDAP instance deployed on kubernetes.
```

Comment:
add usage examples of this script as a comment here.

1. example of how to diagnose issues with cdap system services
2. example of how to collect logs for a failing pipeline run

Reply:
Added